tests/integration/components/bs-modal/header-test.js

```diff
@@ -55,7 +55,7 @@ module('Integration | Component | bs-modal/header', function (hooks) {
   test('close button has expected content', async function (assert) {
     await render(hbs`<BsModal::Header @title="Header" />`);
 
-    assert.dom(`.modal-header button.${closeButtonClass()}`).hasText(versionDependent('×', ''));
+    assert.dom(`.modal-header button.${closeButtonClass()}`).hasText(versionDependent('&times;', ''));
```

Comment:
Reset this change to fix tests.

```suggestion
    assert.dom(`.modal-header button.${closeButtonClass()}`).hasText(versionDependent('×', ''));
```